src/styles/index.scss

```diff
@@ -1,3 +1,108 @@
+@import "./variables";
+@import "./stars.scss";
+
 body {
   margin: 0;
+  font-family: "Roboto", "Open Sans", sans-serif;
+  color: #060b35;
+
+  h1,
+  h2,
+  p {
+    margin: 0;
+  }
+
+  .product-card {
+    width: 200px;
+    padding: 32px 16px 16px;
+    border: 1px solid #f3f3f3;
+    border-radius: 5px;
+    box-sizing: border-box;
```

Comment:
it s better to apply this property to all elements (use * selector)
when all elements have the same `box-sizing` value, their behavior becomes more predictable